src/breadcrumb/breadcrumb.scss

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-breadcrumb {
```

Comment:
I think the whole component should be named breadcrumbs plural

Reply:
Renaming to breadcrumbs / plural in next push.